Assets/Scripts/EnemySystem/ScriptableObjects/EnemyAttackStrategies/BabeAttackStrategy.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Player;
+using UnityEngine;
+
+namespace EnemySystem.ScriptableObjects.EnemyAttackStrategies
+{
+    [CreateAssetMenu(fileName = "BabeAttackStrategy", 
+        menuName = "ScriptableObjects/EnemyAttack/Babe", order = 3)]
+    public class BabeAttackStrategy : EnemyAttackStrategy
+    {
+        private float rangeOfMeleeAttack = 2f;
```

Comment:
It might be a good idea to have these as `[SerializeField]`

Reply:
Unless the size of babe changes, this should always stay the same. I don't think making it SerializeField is necessary here

Reply:
Sounds good!